kubespawner/spawner.py

```diff
@@ -55,11 +55,6 @@ class PodReflector(ResourceReflector):
 
     kind = "pods"
 
-    # The default component label can be over-ridden by specifying the component_label property
-    labels = {
-        'component': 'singleuser-server',
-    }
-
```

Comment:
This is an override of the base class ResourceReflector definition of labels, which is now being passed via `_start_reflector(labels=...)` instead. Presumably, the PodReflector or other base class then should have a constructor accepting that keyword argument...

I concluded that the class inheritance is like this: PodReflector -> ResourceReflector -> LoggingConfigurable -> Configurable -> HasTraits, and that HasTraits does indeed consume it. Okay, this makes sense then!

This is the kind of investigation I felt was needed, making this change something I appreciated having in a standalone PR.

Reply:
I used this as a reference:
https://github.com/jupyterhub/kubespawner/blob/c6bb2a1e3a0ddec6044b4ef8cb31f8fa50571c7c/kubespawner/proxy.py#L271-L279

Reply:
I appreciated that! It made us consistent, something I also checked for when reviewing this.